# --- ROCK-COPYRIGHT-NOTE-BEGIN ---
# 
# This copyright note is auto-generated by ./scripts/Create-CopyPatch.
# Please add additional copyright information _after_ the line containing
# the ROCK-COPYRIGHT-NOTE-END tag. Otherwise it might get removed by
# the ./scripts/Create-CopyPatch script. Do not edit this copyright text!
# 
# ROCK Linux: rock-src/package/*/glibc23/glibc-2.2.2-test-lfs-timeout.patch
# Copyright (C) 1998 - 2003 ROCK Linux Project
# 
# This program is free software; you can redistribute it and/or modify
# it under the terms of the GNU General Public License as published by
# the Free Software Foundation; either version 2 of the License, or
# (at your option) any later version. A copy of the GNU General Public
# License can be found at Documentation/COPYING.
# 
# Many people helped and are helping developing ROCK Linux. Please
# have a look at http://www.rocklinux.org/ and the Documentation/TEAM
# file for details.
# 
# --- ROCK-COPYRIGHT-NOTE-END ---


set the timeout for Reiserfs to 120 seconds

--- ./io/test-lfs.c~	Fri Feb  9 18:04:07 2001
+++ ./io/test-lfs.c	Sat Feb 17 04:30:18 2001
@@ -34,7 +34,7 @@
 #define PREPARE do_prepare
 
 /* We might need a bit longer timeout.  */
-#define TIMEOUT 20 /* sec */
+#define TIMEOUT 120 /* sec */
 
 /* This defines the `main' function and some more.  */
 #include <test-skeleton.c>
